lib/sequel/jdbc/as400.rb

```diff
@@ -59,7 +59,6 @@ def begin_transaction(conn, opts = OPTS)
 
       # Dataset class for AS400 datasets accessed via JDBC.
       class Dataset < JDBC::Dataset
-        include EmulateOffsetWithRowNumber
```

Comment:
is `EmulateOffsetWithRowNumber` used after this?

Reply:
Not as far as I can see, (and it shouldn't since we don't want that behavior)

Reply:
ah, you are asking if the require is still needed